src/main/groovy/com/bmuschko/gradle/docker/tasks/DockerInfo.groovy

```diff
@@ -37,5 +41,9 @@ class DockerInfo extends AbstractDockerRemoteApiTask {
         logger.quiet "NGoroutines          : $info.nGoroutines"
         logger.quiet "Swap Limit           : $info.swapLimit"
         logger.quiet "Execution Driver     : $info.executionDriver"
+
+        if (onNext) {
+            onNext.call(info)
+        }
```

Comment:
Can you add `else` here and move logger statements into it?

Reply:
Oh... so display Or onNext but not both.  Hmm... I kind if liked still having the logger, but that might be a matter of personal taste.  I suppose one can always manually log from within the onNext.

Reply:
Done.